courses/free-courses-de.md

```diff
@@ -11,9 +13,21 @@
 * [Python](#python)
 * [Rust](#rust)
 * [Spieleentwicklung](#spieleentwicklung)
+* [SQL](#sql)
 * [TypeScript](#typescript)
 
 
+### Ansible
+
+* [Ansible Tutorial](https://www.redhat.com/de/topics/automation/learning-ansible-tutorial) - RedHat Tutorial für Ansible
```

Comment:
It looks like this citation should just be `RedHat`
```suggestion
* [Ansible Tutorial](https://www.redhat.com/de/topics/automation/learning-ansible-tutorial) - RedHat
```

Reply:
updated 👍